src/checkstyle/checkstyle.xml

```diff
@@ -36,7 +36,14 @@
         <module name="AvoidStarImport"/>
         <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
         <module name="RedundantImport"/>
-
+        <module name="ImportOrder">
+            <property name="groups" value="java,javax,net.logstash"/>
```

Comment:
remove javax

I think one group for java and javax without a separator between is sufficient.

Reply:
Hmmm... I can have checkstyle putting both `java.*` and `javax.*` in the same group with a regular expression - but IDE like Eclipse doesn't like it :-(

The project isn't using any `javax.*` stuff anyway so, either:
- I remove the `javax` group and let them fall into the "other" category
- I keep it asis with possibly two "java" groups at the top - "just in case..." (project is not using any javax stuff anyway)

Reply:
Let's remove javax.

A lot of that namespace is slowly being replaced with jakarta anyway.

Reply:
I was wrong... we have a few `javax.net.ssl.*` stuff... removing the `javax` group gives something like this:

```
import java.io.BufferedOutputStream;
import java.io.IOException;
import java.io.InputStream;
import java.io.OutputStream;
import java.net.InetAddress;
import java.net.InetSocketAddress;
import java.net.Socket;
import java.net.SocketTimeoutException;
import java.net.UnknownHostException;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Formatter;
import java.util.List;
import java.util.Optional;

import net.logstash.logback.appender.destination.DelegateDestinationConnectionStrategy;
import net.logstash.logback.appender.listener.TcpAppenderListener;
import net.logstash.logback.encoder.SeparatorParser;
import net.logstash.logback.encoder.StreamingEncoder;

import ch.qos.logback.core.net.ssl.ConfigurableSSLSocketFactory;
import ch.qos.logback.core.net.ssl.SSLConfigurableSocket;
import com.lmax.disruptor.EventHandler;
import com.lmax.disruptor.LifecycleAware;
import com.lmax.disruptor.RingBuffer;
import javax.net.SocketFactory;
import javax.net.ssl.SSLContext;
import javax.net.ssl.SSLSocket;
import javax.net.ssl.SSLSocketFactory;
```

Do you still like it? Personally I'm more in favour of a `javax` group at the top below `java`.. matter of taste ;-)
Up to you...

Reply:
Ok.  I was thinking it was prefix based, and that java would match java and javax.  But since that's not the case, we can keep it.